src/mme-app/actionHandlers/attachActionHandlers.cpp

```diff
@@ -495,6 +500,8 @@ ActStatus ActionHandlers::process_sec_mode_resp(SM::ControlBlock& cb)
 	else
 	{
 		log_msg(LOG_INFO, "Sec mode failed. UE %d", ue_ctxt->getContextID());
+		//procedure_p->setMmeErrorCause(secModeRespFailure_c);
```

Comment:
Could you please add a comment here.. Looks like we need to do something which we are not doing now.. Atleast adding TODO comment will be helpful for other readers.

Reply:
Uncommented this line.